.github/workflows/r_valgrind.yml

```diff
@@ -0,0 +1,30 @@
+name: R valgrind tests
+
+on:
+  pull_request_review_comment:
+    types: [created]
+
+jobs:
+  test-r-valgrind:
+    name: r-package (ubuntu-latest, R-devel, valgrind)
+    if: github.event.comment.body == '/gha run r-valgrind-tests' && contains('OWNER,MEMBER,COLLABORATOR', github.event.comment.author_association)
```

Comment:
Maybe just `r-valgrind` for the shortness?

Reply:
sure

Reply:
changed in https://github.com/microsoft/LightGBM/pull/3443/commits/b03d8633ed919e919da5f4c526bfabd0c7e85d0e